Observer/NewTescoPaymentObserver.php

```diff
@@ -0,0 +1,99 @@
+<?php
+namespace Omise\Payment\Observer;
+
+use Magento\Framework\Event\ObserverInterface;
+use Magento\Payment\Model\InfoInterface;
+use Magento\Framework\Event\Observer;
+
+class NewTescoPaymentObserver implements ObserverInterface
```

Comment:
@jacstn I think the classname is a bit ambiguous with the word `New`.
How about just simply `TescoPaymentObserver` or `TescoBillPaymentObserver`?